Makefile

```diff
@@ -78,6 +78,11 @@ get-test-deps: ## Install test dependencies
 	$(GO_NOMOD) get github.com/axw/gocov/gocov
 	$(GO_NOMOD) get -u gopkg.in/matm/v1/gocov-html
 
+tidy-deps: ## Cleans up dependencies
```

Comment:
@ccojocar, I added this rule to do the dep cleanup. It makes sure the tooling dependencies which might get removed as part of the `mod tidy` get added as well. WDYT?

Reply:
lgtm